pkg/tso/config.go

```diff
@@ -70,17 +84,60 @@ func NewConfig() *Config {
 	return &Config{}
 }
 
+const (
+	defaultLeaderLease   = int64(3)
+	defaultMaxResetTSGap = 24 * time.Hour
+
+	defaultName              = "TSO"
+	defaultBackendEndpoints  = "http://127.0.0.1:2379"
```

Comment:
should remove `http://` otherwise an error will be reported `too many colons in address`, when using the default value.

Reply:
you mean defaultListenAddr, correct? changed to defaultListenAddr = "127.0.0.1:2381"